gcovr/tests/shared_lib/Makefile

```diff
@@ -40,9 +40,9 @@ run: txt xml html sonarqube coveralls
 
 coverage.json:
 ifeq ($(BASE_OS),MSYS_NT)
-	PATH="`pwd`/lib:${PATH}" testApp/test/a.out
+	PATH="$(dir $(lastword $(MAKEFILE_LIST)))lib:${PATH}" testApp/test/a.out
```

Comment:
Why doesn't `pwd` work here? Is this because of weird behaviour under Windows?

Using Makefile builtins seems more fragile, in particular because this here might result in relative paths like `PATH=lib:${PATH}`. Could perhaps be clarified as `$(dir $(abspath $(lastword $(MAKEFILE_LIST))))`?

Now that I see it, I'm not sure if `${PATH}` is correct (expands the variable in make) or whether `$$PATH` would be better (expands the variable in shell).

Reply:
This was a try because the dll isn't found in Windows. I rhought that pwd might not work on GitHub with windows.
Two other tests also fail. The problem in CMake I'll discuss tomorrow with a colleague of mine.